### DISCUSSION TITLE
OpenThread UDP send BLOG

### DISCUSSION THREAD

**Discussion by tcpipchip** [openthread/openthread#7469] [Show and tell]
https://nina-b3-openthread.blogspot.com/2022/03/nina-b302-transmitindo-dados-via-udp.html

Enjoy!!!!

**Comment by jwhui** [comment]
@tcpipchip , thanks for sharing!

**Comment by tcpipchip** [comment]
it´s a pleasure!

Btw, does someone have examples about Thread UDP SEND  on VISUAL CODE + NRFCONNECT + RTOS  to me to test ?

**Reply by jwhui** [reply]
nRF Connect SDK is built on Zephyr. You can look through the [nRF Connect SDK documentation](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/openthread/cli/README.html) for OpenThread CLI examples.

**Comment by tcpipchip** [comment]
That’s is only to CLI

I need something similar to that segger example on blog, but for zephyr


Enviado do meu iPhone

Em 9 de mar. de 2022, à(s) 16:35, Jonathan Hui ***@***.***> escreveu:

﻿

nRF Connect SDK is built on Zephyr. You can look through the nRF Connect SDK documentation<https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/openthread/cli/README.html> for OpenThread CLI examples.

—
Reply to this email directly, view it on GitHub<https://github.com/openthread/openthread/discussions/7469#discussioncomment-2326552>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ACXUZTAEKFRRXCGIVGPZX3TU7D4RJANCNFSM5QKIYM2A>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Comment by tcpipchip** [comment]
https://nina-b3-openthread.blogspot.com/2022/03/nina-b302-transmitindo-dados-via-udp.html
